### ISSUE TITLE
Type `area` fillColor with rgba does not work in Microsoft Edge

### ISSUE THREAD

**Issue by ppcano** [highcharts/highcharts#5267]
#### Expected behaviour
![screen shot 2016-04-29 at 12 03 40](https://cloud.githubusercontent.com/assets/825430/14913315/7600ae9c-0e02-11e6-8c02-4f6b7ec063f9.png)

#### Actual behaviour

![screen shot 2016-04-29 at 11 52 00](https://cloud.githubusercontent.com/assets/825430/14913321/7da9591e-0e02-11e6-9477-26aed1220373.png)

#### Live demo of the issue

```
          marker: {
            enabled: true,
            symbol: 'circle',
            radius: 4,
            lineWidth: 3,
            lineColor: color,
            fillColor: 'white'
          },
          fillColor: {
            linearGradient: {
              x1: 0,
              y1: 0,
              x2: 0,
              y2: 1
            },
            stops: [
                 [0, Highcharts.Color(color).setOpacity(0.2).get('rgba')],
                 [0.4, Highcharts.Color(color).setOpacity(0.1).get('rgba')],
                 [0.6, Highcharts.Color(color).setOpacity(0).get('rgba')],
                 [1, Highcharts.Color(color).setOpacity(0).get('rgba')]
            ]
          },
          dataGrouping: {
            approximation: approximation,
            enabled: true,
            smoothed: true,
            groupPixelWidth: 35
          },
          id: modelSerie.get('id'),
          yAxis: modelSerie.get('id'),
          type: 'area',
          data: [],
          name: name


```
#### Affected browser(s)

Microsoft Edge 13

**Comment by TorsteinHonsi** [comment]
Thanks for reporting!

Can you provide a live reproduction? I tried to [set it up in jsFiddle](http://jsfiddle.net/highcharts/wk0nsczy/), but it works well there.

**Comment by ppcano** [comment]
We are using Highstock and I have tried to replicate our setup in this [jsFiddle](http://jsfiddle.net/ppcanodehuelva/xeLqo2nx/4/).

Unfortunately, I have not been able to reproduce it.

It only happens in Microsoft Edge 13.

**Comment by TorsteinHonsi** [comment]
Ok, are you able to reproduce it outside jsFiddle, somewhere public so we can see it in action?

**Comment by ppcano** [comment]
@TorsteinHonsi, No public now, but you contact me at pepe@loadimpact.com to provide you a testing environment.

**Comment by jon-a-nygaard** [comment]
@ppcano Hi, are you still experiencing this issue?
I tested your setup on Microsoft Edge 13 in Windows 10 and it seems to work fine. 
![Screenshot](https://s12.postimg.org/6peumqngd/screenshot.png)

**Comment by dawid84** [comment]
Hi, we believe this is related to (or the same as) an issue we are having in our Angular 2 app when attempting to use patterns as the fill and determined the inclusion of a `<base>` tag is the cause.

The issue is reproduced in these two fiddles:
[http://jsfiddle.net/es06yqar/1](http://jsfiddle.net/es06yqar/1)
[http://jsfiddle.net/n1tyxL9f](http://jsfiddle.net/n1tyxL9f)

**Comment by jon-a-nygaard** [comment]
@dawid84 Thank you for the feedback. I have tested in Edge 38, and I can see that the issue has been fixed in our latest version 5.0.0.

**Comment by ppcano** [comment]
@jon-a-nygaard I confirm, it works in version 5.0.0
